.github/workflows/ci.yml

```diff
@@ -94,11 +94,11 @@ jobs:
 
   test-on-linux:
     name: Test on Linux
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
NOTE: `ubuntu-latest` runner has been changed to 22.04, which is not supporting `Swift < 5.7`